plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryCliProvider.kt

```diff
@@ -93,43 +102,37 @@ internal object SentryCliProvider {
         }
     }
 
-    internal fun searchCliInResources(resourcePath: String): String? {
-        val resourceURL = javaClass.getResource(resourcePath)
-        val resourceFile = resourceURL?.let { File(it.file) }
```

Comment:
I'm really not sure why this was done this way 😅 Shouldn't it be fine to just null check the `resourceURL`?

Reply:
yeah I also don't know, null check should be fine I guess 👍